draft-bbs-signatures.md

```diff
@@ -450,6 +450,38 @@ Procedure:
 How a signature is to be encoded is not covered by this document. (TODO perhaps add some additional information in the appendix)
 (FIXME: Encoding out of scope OK, but wire format is required for test vectors. Unless this document is not normative.)
 
+### Alternative Description of SpkGen
+The following provides a higher-level description of SpkGen. Let the prover be in possession of a BBS signature `(A, e, s)` with `A = b * (1/(e + Sk))` where `Sk` the signer's secret key and,
+
+    b = P1 + h0 * s + h[1] * msg_1 + ... + h[L] * msg_L
+
+(without loss of generality we assume that the messages and generators are indexed from 0 to L). Let `(i1,...,iR)` be the indexes of generators corresponding to messages the prover wants to disclose and `(j1,...,jU)` be the indexes corresponding to undisclosed messages (i.e., `(j1,...,jU) = [L] \ (i1,...,iR)`). To prove knowledge of a signature on the disclosed messages, work as follows,
+
+- Randomize the signature `(A, e, s)`, by taking uniformly random `r1`, `r2` in [1, r-1], and calculate, 
```

Comment:
Assuming r is as defined in terminology? If so, no need to change anything

Reply:
Yes, r is as defined in the Notation section (the prime order of G1 and G2).